test/hotspot/jtreg/serviceability/sa/TestSysProps.java

```diff
@@ -65,7 +65,7 @@ public static void countProps(String[] propLines, int expectedCount, String cmdN
             }
         }
         if (numProps != expectedCount) {
-            throw new RuntimeException("Wrong number of " + cmdName + " properties: " + numProps);
+            throw new RuntimeException("Wrong number of " + cmdName + " properties: " + numProps + " Expected: " + expectedCount);
```

Comment:
I think it would be good to add parenthesis around the extra output you added.

Reply:
This was an accidental leftover from debugging, I didn't intend for this to be part of the change. I should revert this since it's beyond the scope of this change.